developerportal/templates/organisms/featured.html

```diff
@@ -23,13 +23,13 @@ <h2>Featured</h2>
         {% for featured_item in featured %}
           {% if not forloop.first %}
             {% with featured_item as block %}
-              {% if block.block_type == 'post' or block.block_type == 'event' %}
+              {% if block.block_type == 'post' or block.block_type == 'event' or block.block_type == 'video' %}
```

Comment:
I'm not a jinja-ninja, but can we reuse the very similar block from above? Usually I only go for reuse when things are used in more than 2 places but this looks complex enough (to me) to warrant reuse

Reply:
Yeah, that's an astute call - one thing in the backlog is to move to Jinja2 (from Django Template Language) to allow for macros and easier block reuse. This is something the original dev team recommended and i def agree with.

For now, given we are very very likely going to update these templates post-launch, I'm deliberately avoiding refactoring significantly, though.